thirdparty/build-thirdparty.sh

```diff
@@ -564,6 +564,14 @@ build_gtest() {
     ${BUILD_SYSTEM} install
 }
 
+# xxhash
+build_xxhash() {
+    check_if_source_exist $XXHASH_SOURCE
+    mkdir -p $TP_INCLUDE_DIR
+    # xxhash.h is in the root directory of xxHash source
+    cp $TP_SOURCE_DIR/$XXHASH_SOURCE/xxhash.h $TP_INCLUDE_DIR/
```

Comment:
```suggestion
    mkdir -p $TP_INCLUDE_DIR/xxhash && cp $TP_SOURCE_DIR/$XXHASH_SOURCE/xxhash.h $TP_INCLUDE_DIR/xxhash/
```
How about moving it into a dedicated directory?